### PR TITLE
jreleaser: update to version v0.10.0

### PR DESCRIPTION
**Pull request by aalmiray**

### CODE REVIEW

File: Food/jreleaser.lua [hunk 1]
```diff
@@ -33,17 +36,57 @@ food = {
...
             arch = "amd64",
             url = "https://github.com/" .. name .. "/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-standalone-" .. version .. "-windows-x86_64.zip",
-            sha256 = "5b9b7616b948aebeaef158cc62d4a5c512ecce5e8f1dd5d49758e1a341177f35",
+            sha256 = "3f3948542208d297d794342e109df3f0c1c023d4142024688125a2aa9b9c7a72",
             resources = {
                 {
                     path = name .. "-standalone-" .. version .. "-windows-x86_64\\bin\\" .. name .. ".bat",
-                    installpath = "bin\\" .. name .. ".bat"
+                    installpath = "bin\\" .. name .. ".bat",
+                    executable = true
```
Comment (bacongobbler) [inline Food/jreleaser.lua:87]: Not necessary for windows targets

File: Food/jreleaser.lua [hunk 2]
```diff
@@ -33,17 +36,57 @@ food = {
...
+        {
+            os = "windows",
+            arch = "arm64",
+            url = "https://github.com/" .. name .. "/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-standalone-" .. version .. "-windows-aarch64.zip",
+            sha256 = "bd7b4c780ee4f5a3c3ad8ea5f6eb9a74fc3d50ec07a59b39d40918bfebc31a11",
+            resources = {
+                {
+                    path = name .. "-standalone-" .. version .. "-windows-aarch64\\bin\\" .. name .. ".bat",
+                    installpath = "bin\\" .. name .. ".bat",
+                    executable = true
```
Comment (bacongobbler) [inline Food/jreleaser.lua:74]: Not necessary for windows targets

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: Food/jreleaser.lua (hunk 1 above)
Comment (aalmiray) [reply Food/jreleaser.lua:87]: I noticed `executable = true` was not specified before but when I looked at other food entries I think I saw it explicitly set. I can remove it of course.

Comment (aalmiray) [reply Food/jreleaser.lua]: Removed and squashed.

**Review by bacongobbler — APPROVED** [review]

**Comment by aalmiray** [comment]
Thanks!

**Pull request merged**